charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -24,6 +24,9 @@ spec:
         {{- end }}
       labels:
         {{- include "scalardb-cluster.selectorLabels" . | nindent 8 }}
+        {{- if eq .Values.global.platform "azure" }}
+        azure-extensions-usage-release-identifier: {{ .Release.Name }}
+        {{- end }}
```

Comment:
This configuration is necessary to charge fees based on the number of pods.
https://learn.microsoft.com/en-us/partner-center/marketplace-offers/azure-container-technical-assets-kubernetes#make-updates-based-on-your-billing-model